### DISCUSSION TITLE
Sometimes browser window covers taskbar and windows start doesn't respond

### DISCUSSION THREAD

**Discussion by steveja42** [valinet/ExplorerPatcher#1827] [General]
Sometimes when I wake the computer, the browser windows (and vscode) cover the taskbar when maximized and also there is no response  when clicking on windows start icon or pressing the windows key.  If I restart file explorer from the properties dialog the problem is temporarily solved until it happens again.

Config:
Explorer Patcher 22621.1992.56.1
Windows 11 build 22621.2134
Microsoft Surface 7 plus with 2 external monitors attached, one  of which is the primary monitor.

**Comment by pyrates999** [comment]
Does this happen every time you wake the computer up from sleep mode?

**Comment by steveja42** [comment]
No, not every time. I haven't isolated what causes it.

**Comment by pyrates999** [comment]
Do you go into hibernation mode where the system is completely off or in sleep mode?

**Comment by steveja42** [comment]
sleep mode

**Comment by pyrates999** [comment]
Might be related to the 2 external monitors you have attached.  Disconnect them and then go into sleep mode to see if this still occurs.

**Comment by steveja42** [comment]
It doesn't seem to happen  when the external monitors are not attached.

**Reply by pyrates999** [reply]
Then I'd say it's a bug in windows, not explorer patcher.

**Comment by yawnee** [comment]
After the recent windows 11 update... I'm having the same issue. Can't open start menu

**Reply by pyrates999** [reply]
Check that symbols were downloaded: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by yawnee** [reply]
Not sure about the symbols. But I reverted the update and it works now.

**Reply by pyrates999** [reply]
Ok.  Please learn how to check for symbols being downloaded using the above link as you will need it with every new windows update.
